packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -315,8 +246,117 @@ import { name, version } from '../package'
+  // Boolean flag
+  let hasPassedEngineCheck = null
+  // Array of strings
+  let engineErrorLog = []
+  // Docs link for engine errors
+  const engineErrorDocsLink = terminalLink(
+    'Tutorial - Prerequisites',
+    'https://redwoodjs.com/docs/tutorial/chapter1/prerequisites'
+  )
+
+  await new Listr(
+    [
+      {
+        title: 'Checking node and yarn compatibility',
+        skip: () => {
+          if (yarnInstall === false) {
+            return 'Warning: skipping check on request'
+          }
+        },
+        task: () => {
+          return new Promise((resolve) => {
+            const { engines } = require(path.join(templateDir, 'package.json'))
+
+            // this checks all engine requirements, including Node.js and Yarn
+            checkNodeVersion(engines, (_error, result) => {
+              if (result.isSatisfied) {
+                hasPassedEngineCheck = true
+                return resolve()
+              }
+              const logStatements = Object.keys(result.versions)
+                .filter((name) => !result.versions[name].isSatisfied)
+                .map((name) => {
+                  const { version, wanted } = result.versions[name]
+                  return style.error(
+                    `${name} ${wanted} required, but you have ${version}`
+                  )
+                })
+              engineErrorLog = logStatements
+              hasPassedEngineCheck = false
+              return resolve()
+            })
+          })
+        },
+      },
+    ],
+    { rendererOptions: { clearOutput: true } }
+  ).run()
+
+  // Show a success message if required engines are present
+  if (hasPassedEngineCheck === true) {
+    console.log(style.success(`✔️ Compatability checks passed.`))
+  }
+
+  // Show an error and prompt if failed engines check
+  if (hasPassedEngineCheck === false) {
+    console.log(style.error(`✖️ Compatability checks failed.`))
+    console.log(`${engineErrorLog.join('\n')}`)
+    console.log(
+      style.error(
+        `\nThis might make your RedwoodJS project incompatible with some deploy targets.`
+      )
+    )
+    console.log(style.header(`\nRelated documentation:`))
+    console.log(style.warning(`${engineErrorDocsLink}\n`))
+    const response = await prompt({
+      type: 'select',
+      name: 'override-engine-error',
+      message: 'How would you like to proceed?',
+      choices: [
+        // See https://github.com/enquirer/enquirer#defining-choices
+        // Does not seem to be working properly with the values as strings or booleans
+        // Possibly an upstream problem in Listr2
+        // { name: 'Override error and continue install', value: 'true' },
+        // { name: 'Quit install', value: 'false' },
+        'Override error and continue install',
+        'Quit install',
+      ],
+      initial: 0,
+    })
+    // TODO improve this logic if bug above can be fixed
```

Comment:
I don't think it's too bad the way you've done it here.
But if you wanted to change it, maybe you could do `value: 1` and `value: 0` and then `if (!response['override-engine-error']) {`

Reply:
There is something weird going on with the `value` key not being properly passed through to the response/result - I tried a few different options here but the only thing I can think of is that the value isn't being properly read by Listr in their implementation of the Enquirer api.

Reply:
Hey @ehowey I had a quick look at this. I don't think it's to do with listr given that you're using prompt directly from enquirer and not using `task.prompt` from within the listr tasks.

I was confused as to why it was never returning the `value` and always just the `name`. I had a look for examples within enquirer and found:
https://github.com/enquirer/enquirer/blob/36785f3399a41cd61e9d28d1eb9c2fcd73d69b4c/examples/select/select-values.js#L1-L17

The thing I noticed is that they avoid the `name` within the choice option and use `message` and `value` instead. I tried that here and it appears to work. So by changing:
```ts
{ name: 'Override error and continue install', value: 'true' }
```
to
```ts
{ message: 'Override error and continue install', value: 'true' }
```
Then I can have the response object have the `value` e.g.:
```ts
{ 'override-engine-error': 'true' }
```

There is still the limitation that values must be strings and it didn't like an empty string either - it returned the `name` in that case if I remember correctly.

Hope this helps.

Reply:
Great catch Josh, thanks.  That did work! But after testing I ended up leaving it as is.  The answer is actually shown to the user in the console (see pics), and doesn't create a good ux showing them false.

With false as the value:
<img width="623" alt="Screen Shot 2022-10-27 at 8 59 17 PM" src="https://user-images.githubusercontent.com/204841/198487517-295ffbaa-14b1-40d7-9e5f-55f67ff15f6d.png">

With just "Quit install" as it was:
<img width="640" alt="Screen Shot 2022-10-27 at 9 01 08 PM" src="https://user-images.githubusercontent.com/204841/198487653-2b990dc1-10ec-4752-b2c1-1ce10fbce815.png">